examples/case_studies/hierarchical_partial_pooling.ipynb

```diff
@@ -4,11 +4,12 @@
    "cell_type": "markdown",
```

Comment:
<p>There should be a level 2 markdown secrion title right before the code cell with the watermark. https://docs.pymc.io/en/latest/contributing/jupyter_style.html#watermark</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/393/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Thanks for the review! Does it look better now?</p>